docs/Advanced-Usage/FireAxe-Partitioning-onto-Multiple-FPGAs/index.rst

```diff
@@ -0,0 +1,18 @@
+FireAxe Partitioning onto Multiple FPGAs
+=============================================
+
+Although the FPGA capacity has become large enough to simulate large SoCs, there
+still are cases when your design does not fit on a single FPGA.
+When the design contains multiple duplicate modules, you should refer to the
+:ref:`Multithreading<FAME-5>` section first. When the design doesn't fit even with :ref:`Multithreading<FAME-5>`,
+FireAxe can be used with :ref:`Multithreading<FAME-5>` to provide extra simulation capacity.
```

Comment:
This implies that FireAxe is only useful w/ the multi-threading opt. Instead, I would re-word this to focus on FireAxe as the main feature (and mention that this is still compatible with the other features).